src/org/python/core/RegistryKey.java

```diff
@@ -0,0 +1,297 @@
+package org.python.core;
+
+/**
+ *
+ * Supported registry keys and their usage. 
+ *
+ * Boolean properties are set with the 
+ * String values {@code true} or {@code yes} for true, or {@code false} or 
+ * {@code no} for false.
+ *
+ */
+public class RegistryKey {
+
+    private RegistryKey(){}
+
+    /**
+     *
+     * {@code python.cachedir} defines the directory to use for caches 
+     * (currently just package information).
+     * This directory should be writable by the user. If this is an absolute 
+     * path it is used as given, otherwise it is interpreted relative to 
+     * sys.prefix (typically the directory of this file).
+     */
+    public static final String PYTHON_CACHEDIR = "python.cachedir";
+
+    /**
+     *
+     * Setting {@code python.cachedir.skip} to true disables the package scan 
+     * for the cachedir (as defined by {@code python.cachedir} or a default). 
+     * Please be aware that disabling this will break importing from 
+     * java packages.
+     */
+    public static final String PYTHON_CACHEDIR_SKIP = "python.cachedir.skip";
+
+    /**
+     *
+     * {@code python.cpython2} is the name of the CPython executable. 
```

Comment:
Maybe this should specify that it must be 2.7, i.e. not s.5.x or earlier. This is important because of bytecode format.

Reply:
Added